api_app/analyzers_manager/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from django.contrib.postgres import fields as postgres_fields
+
+
+class AnalyzerReport(models.Model):
+    STATUS_CHOICES = (
+        ("pending", "pending"),
+        ("running", "running"),
+        ("failed", "failed"),
+        ("success", "success"),
+    )
+
+    analyzer_name = models.CharField(max_length=128)
+    job = models.ForeignKey(
+        "api_app.Job", related_name="analyzer_reports", on_delete=models.CASCADE
+    )
+
+    status = models.CharField(
+        max_length=50,
+        choices=STATUS_CHOICES,
```

Comment:
After the `Statuses` enum, this would be changed to `[(s.name, s.name) for s in Statuses]`